include/oneapi/dpl/pstl/omp/util.h

```diff
@@ -152,36 +152,46 @@ __process_chunk(const __chunk_metrics& __metrics, _Iterator __base, _Index __chu
 
 namespace __detail
 {
-struct __get_num_threads
+
+// Workaround for VS 2017: declare an alias to the CRTP base template
+template <typename _ValueType, typename... _Args>
+struct __enumerable_thread_local_storage;
+
+template <typename... _Ts>
+using __etls_base = __utils::__enumerable_thread_local_storage_base<__enumerable_thread_local_storage, _Ts...>;
+
+template <typename _ValueType, typename... _Args>
+struct __enumerable_thread_local_storage : public __etls_base<_ValueType, _Args...>
```

Comment:
```suggestion
struct __enumerable_thread_local_storage : __etls_base<_ValueType, _Args...>
```

Reply:
I prefer to be explicit, since it's important there that the API of the base class is publicly available in the derived class.

Reply:
ok